src/commands/rank.ts

```diff
@@ -7,13 +7,26 @@ const RankCommand: CommandFile = {
   data: new SlashCommandBuilder()
     .setName("leaderboard")
     .setDescription("See who's the greatest racist")
+    .addIntegerOption((option) =>
+      option
+        .setName("limit")
+        .setDescription("The number of people on the list (max: 10)")
+        .setRequired(false)
+    )
+
     .toJSON(),
   async execute(interaction, client, db) {
+    let limitOption = Number(
```

Comment:
make this a `const` value
```suggestion
    const limitOption = Number(
```